models/addresses/sanctioned_addresses_ofac.sql

```diff
@@ -0,0 +1,51 @@
+{{config(alias='sanctioned_addresses')}}
```

Comment:
I suggest changing this model's name to models/addresses/ethereum/addresses_ethereum_ofac_sanctionned.sql